extensions/wikia/ExactTargetUpdates/client/adapters/UserEditsAdapter.class.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Wikia\ExactTarget;
+
+class UserEditsAdapter {
+
+	var $edits = [ ];
+
+	public function __construct( $result ) {
+		if ( $result->Properties instanceof \stdClass ) {
+			$this->extractUserWikiEdits( $result->Properties->Property );
```

Comment:
I never found a case where just Properties were present without Property, I'm happy for it to fail here if this happen.

Do you think that's OK or better error handling should be provided here? I think default one should say just enough.